packages/jetpack-blocks/webpack.config.js

```diff
@@ -0,0 +1,151 @@
+/**
+ **** WARNING: No ES6 modules here. Not transpiled! ****
+ */
+/* eslint-disable import/no-nodejs-modules */
+
+/**
+ * External dependencies
+ */
+const _ = require( 'lodash' );
+const path = require( 'path' );
+const fs = require( 'fs' );
+const webpack = require( 'webpack' );
+const CopyWebpackPlugin = require( 'copy-webpack-plugin' );
+const DuplicatePackageCheckerPlugin = require( 'duplicate-package-checker-webpack-plugin' );
+const FileConfig = require( '@automattic/calypso-build/webpack/file-loader' );
+const Minify = require( '@automattic/calypso-build/webpack/minify' );
+const SassConfig = require( '@automattic/calypso-build/webpack/sass' );
+const TranspileConfig = require( '@automattic/calypso-build/webpack/transpile' );
+const wordpressExternals = require( '@automattic/calypso-build/webpack/wordpress-externals' );
+
+/**
+ * Internal dependencies
+ */
+const cacheIdentifier = require( '../../server/bundler/babel/babel-loader-cache-identifier' ); // FIX ME
```

Comment:
This can probably be removed but I suppose that can be a separate PR as well.

Convo p1552997442520300-slack-jetpack-gutenberg

Reply:
I'm digging into this right now. Will report back…

Reply:
Let's drop it.

[Babel-loader default](https://github.com/babel/babel-loader/blob/5da4fa0c673474ab14bc829f2cbe03e60e437858/src/index.js#L178-L182):

```js
      const cacheIdentifier = JSON.stringify({
        options,
        "@babel/core": transform.version,
        "@babel/loader": pkg.version,
      });
```

Our implementation:
https://github.com/Automattic/wp-calypso/blob/7a09dd2cf8219188bdc9c3e668cc8c34108169bc/server/bundler/babel/babel-loader-cache-identifier/index.js#L28-L36

It's essentially the same with the addition of our custom babel async transform which we aren't (I hope) bringing with us.